addons/website_sale_product_configurator/static/src/js/website_sale_options.js

```diff
@@ -80,7 +80,7 @@ publicWidget.registry.WebsiteSale.include({
             },
             success: function (quantity) {
                 if (goToShop) {
-                    var path = window.location.pathname.replace(/shop([\/?].*)?$/, "shop/cart");
+                    var path = "/shop/cart";
```

Comment:
Why this change ?

Reply:
Before, this part was called when we were in the shop so with a URL containing "/shop/". But when we are in a course the URL is not "/shop/" but "/slides/". So when we try to buy a course with the button "Add to Cart" and we click on proceed checkout, we weren't redirect on the cart because of the regex.
I didn't find an utility to keep the regex while we are redirect to the cart anyway in both situation.